pyproject.toml

```diff
@@ -65,7 +65,7 @@ tokio = "tokio:TokioLoopPolicy"
 asphalt = "asphalt.core.cli:main"
 
 [tool.hatch.version]
-path = "src/asphalt/core/__init__.py"
+path = "asphalt/core/__init__.py"
```

Comment:
I'm not giving up on the `src/` layout. If hatch can't work with it, then this is a showstopper issue for me.

Reply:
https://github.com/asphalt-framework/asphalt/pull/74/commits/1d52d54f0ca35a0a91af8f857097ab5f16ac7e82 should do it.